From 8ab032967f60449052c20adc6180c8f295cb37d5 Mon Sep 17 00:00:00 2001
From: Matthew Fatheree <Matthew.Fatheree@belkin.com>
Date: Sat, 19 Apr 2014 00:24:54 +0700
Subject: [PATCH 09/34] CONFIG_UBIFS_FS_XATTR is no longer defined in 3.10

In kernel 3.10, CONFIG_UBIFS_FS_XATTR is no longer defined, we should
remove the "ifdef" to enable ubifs xattr

Signed-off-by: Matthew Fatheree <Matthew.Fatheree@belkin.com>
---
 .../550-ubifs-symlink-xattr-support.patch          |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/target/linux/generic/patches-3.10/550-ubifs-symlink-xattr-support.patch b/target/linux/generic/patches-3.10/550-ubifs-symlink-xattr-support.patch
index b4de071..0235340 100644
--- a/target/linux/generic/patches-3.10/550-ubifs-symlink-xattr-support.patch
+++ b/target/linux/generic/patches-3.10/550-ubifs-symlink-xattr-support.patch
@@ -1,15 +1,13 @@
 --- a/fs/ubifs/file.c
 +++ b/fs/ubifs/file.c
-@@ -1576,6 +1576,12 @@ const struct inode_operations ubifs_syml
+@@ -1576,6 +1576,10 @@ const struct inode_operations ubifs_syml
  	.follow_link = ubifs_follow_link,
  	.setattr     = ubifs_setattr,
  	.getattr     = ubifs_getattr,
-+#ifdef CONFIG_UBIFS_FS_XATTR
 +	.setxattr    = ubifs_setxattr,
 +	.getxattr    = ubifs_getxattr,
 +	.listxattr   = ubifs_listxattr,
 +	.removexattr = ubifs_removexattr,
-+#endif
  };
  
  const struct file_operations ubifs_file_operations = {
-- 
1.7.9.5

